src/ui/actions/GetTaskToShowLabelsAction.cpp

```diff
@@ -18,6 +18,5 @@ ActionResult GetTaskToShowLabelsAction::execute(const std::shared_ptr<ModelInter
     }
 
     std::vector<Core::TaskEntity> tasks = model->getTasks(*id);
-    tasks.erase(tasks.begin() + 1, tasks.end());
-    return {ActionResult::Status::SUCCESS, tasks};
+    return {ActionResult::Status::SUCCESS, tasks[0]};
```

Comment:
maybe there is a need to check if model returned any tasks by this id?

Reply:
should be OK, but fixed to make it explicit